newm-server/src/main/kotlin/io/newm/server/aws/AwsSecretsManagerConfigLoader.kt

```diff
@@ -0,0 +1,66 @@
+package io.newm.server.aws
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.secretsmanager.AWSSecretsManagerAsyncClientBuilder
+import com.amazonaws.services.secretsmanager.model.GetSecretValueRequest
+import com.typesafe.config.ConfigFactory
+import io.ktor.server.config.ApplicationConfig
+import io.ktor.server.config.ConfigLoader
+import io.ktor.server.config.HoconApplicationConfig
+import kotlinx.serialization.json.Json
+import java.io.File
+
+class AwsSecretsManagerConfigLoader : ConfigLoader {
+    private val secretsManager = AWSSecretsManagerAsyncClientBuilder.standard()
+        .withRegion(Regions.fromName("us-west-2"))
```

Comment:
If the `by inject()` works (previous comment), then you should be able to do:
```
private val regions: Regions by inject()

```
since `Regions` is already being injected.